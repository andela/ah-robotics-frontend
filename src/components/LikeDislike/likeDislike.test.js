import { shallow } from 'enzyme';
import React from 'react';
import LikeDislikeComponent from './index';

describe('Add LikeDislike Component Tests', () => {
  const wrapper = shallow(<LikeDislikeComponent />);

  it('app should render LikeDislikeComponent', () => {
    expect(wrapper.length).toBe(1);
  });

  it('it should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
