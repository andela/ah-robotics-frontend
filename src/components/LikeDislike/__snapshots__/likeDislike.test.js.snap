// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add LikeDislike Component Tests it should match snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
